### PR TITLE
Add EthIndia Community Page

### PR DESCRIPTION
**Pull request by stevenelleman**

### CODE REVIEW

File: apps/frontend/src/features/community/EthIndiaCommunityPage.tsx [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+  useEffect(() => {
+    const fetchInfo = async () => {
+      const { user, session } = await storage.getUserAndSession();
+      if (!user || !session || session.authTokenExpiresAt < new Date()) {
+        toast.error("Please log in to view the leaderboard.");
+        router.push("/");
+        return;
+      }
+
+      const communityIssuer: ChipIssuer = ChipIssuer.ETH_INDIA_2024;
```
Comment (stevenelleman) [inline apps/frontend/src/features/community/EthIndiaCommunityPage.tsx:65]: Updated from Devcon.

File: apps/frontend/src/components/StoreBanner.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ export const StoreBanner = () => {
     // <Link href="/shop" className="flex w-full">
     <div className="flex flex-col gap-1">
       <span className="text-base font-bold font-sans text-label-primary">
-        Museum store
+        Merch store
```
Comment (stevenelleman) [inline apps/frontend/src/components/StoreBanner.tsx:12]: Is "Merch" okay or is it too informal?

File: apps/frontend/src/features/community/EthIndiaCommunityPage.tsx [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+              </a>{" "}
+              servers, who work together to generate a proof you have a valid
+              tap. This means you can delegate ZK proving without sacrificing
+              privacy!
+            </span>
+          </>
+        }
+        leaderboardDetails={leaderboardTapDetails}
+        leaderboardEntries={leaderboardTapEntries}
+        goal={2000}
```
Comment (stevenelleman) [inline apps/frontend/src/features/community/EthIndiaCommunityPage.tsx:250]: Ah updating this.

File: apps/frontend/src/components/StoreBanner.tsx (hunk 2 above)
Comment (RiverRuby) [reply apps/frontend/src/components/StoreBanner.tsx:12]: yep lgtm

**Review by AndrewCLu — APPROVED** [review]

**Pull request merged**